src/platform/bitboxbase/leds.c

```diff
@@ -15,33 +15,69 @@
 #include "leds.h"
 #include "bitboxbase_pins.h"
 
-static void _turn_small_led(int led, bool level)
+#include <stdint.h>
+static uint8_t small_leds[] = {PIN_LED_SMALL(0),
+                               PIN_LED_SMALL(1),
+                               PIN_LED_SMALL(2),
+                               PIN_LED_SMALL(3),
+                               PIN_LED_SMALL(4)};
+
+void turn_small_led(int led, bool level)
 {
     gpio_set_pin_level(led, level);
 }
 
-static void _turn_big_led(int led, bool level)
+static void _turn_big_led_component(uint8_t pin, bool level)
 {
-    gpio_set_pin_level(led, !level);
+    gpio_set_pin_level(pin, !level);
 }
 
-typedef enum { LED_WHITE, LED_RED, LED_GREEN, LED_BLUE } led_color_t;
+static const uint8_t big_leds[2][3] = {{PIN_BLED0_R, PIN_BLED0_G, PIN_BLED0_B},
+                                       {PIN_BLED1_R, PIN_BLED1_G, PIN_BLED1_B}};
 
-static const uint8_t big_leds[] =
-    {PIN_BLED0_R, PIN_BLED0_G, PIN_BLED0_B, PIN_BLED1_R, PIN_BLED1_G, PIN_BLED1_B};
+void turn_big_led(int led, led_color_t color)
```

Comment:
I would like to have something like `enum led_big_t {LED_BIG_LEFT, LED_BIG_RIGHT}` and then take `enum led_big_t` instead of `int` in `turn_big_led`.

Reply:
+1 for left/right LED instead of 0/1. At the same time though, the small LEDs are better indexed by number 0-4, so consistency might be a factor here.

Reply:
I've put in a consistent numbering for both small and big leds, I'd stick to that if everybody agrees.